pkg/secretless/version.go

```diff
@@ -4,7 +4,7 @@ import "fmt"
 
 // Version field is a SemVer that should indicate the baked-in version
 // of the broker
-var Version = "1.4.1"
+var Version = "1.5.0"
```

Comment:
I would not like to bump the version with this PR - please leave at 1.4.1 and we'll bump soon in a separate PR